src/stories/pages/Login/Login.js

```diff
@@ -0,0 +1,121 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { SplitGradient } from '../../layouts/SplitGradient/SplitGradient';
+import { Header } from '../../layouts/Header/Header';
+import { Footer } from '../../layouts/Footer/Footer';
+import { Branding } from '../../components/Branding/Branding';
+import { Main } from '../../layouts/Main/Main';
+import { Section } from '../../components/Section/Section';
+import { FormItem } from '../../components/FormItem/FormItem';
+import { Form } from '../../components/Form/Form';
+import { Button } from '../../components/Button/Button';
+import UserServices from '../../../firebase/CRUDServices/userServices';
+import '../../../firebase/firebase';
+
+/**
+ * Component for login page.
+ *
+ * @component
+ * @param {boolean} isDarkTheme theme color of the page.
+ * @return {object} (
+ *   <Login isDarkTheme={isDarkTheme}>
+ * )
+ */
+
+export const Login = ({ isDarkTheme }) => {
+  const [error, setError] = useState(false);
+  const [username, setUsername] = useState('');
+  const [password, setPassword] = useState('');
+
+
+  const getUser = (e) => {
+    e.preventDefault();
+
+    const userService = new UserServices();
```

Comment:
Line 34 should be _const userService = UserServices.getInstance();_ that way it will trigger the singleton creation process and assign memory to object only if it does not exists, which prevents the program to repeatedly assign and trash memory every time service gets called. Everything else looks good! Thanks for being proactive.

Reply:
@bleachalon can you take a look at this branch? Calling `getInstance()` returns `undefined`.